utility/src/main/java/org/oppia/util/parser/RepositoryGlideModule.kt

```diff
@@ -1,17 +1,25 @@
 package org.oppia.util.parser
 
 import android.content.Context
+import android.graphics.drawable.PictureDrawable
 import com.bumptech.glide.Glide
 import com.bumptech.glide.Registry
 import com.bumptech.glide.annotation.GlideModule
 import com.bumptech.glide.module.AppGlideModule
-import java.io.InputStream
+import com.caverock.androidsvg.SVG
 import org.oppia.util.caching.AssetRepository
+import java.io.InputStream
 
 /** Custom [AppGlideModule] to enable loading images from [AssetRepository] via Glide. */
 @GlideModule
 class RepositoryGlideModule : AppGlideModule() {
+//  override fun registerComponents(context: Context, glide: Glide, registry: Registry) {
+//    registry.prepend(ImageAssetFetcher::class.java, InputStream::class.java, RepositoryModelLoader.Factory())
```

Comment:
Remove this code if not needed

Reply:
I want @BenHenning approval on this. As I have made changes in RepositoryGlideModule. Will this affect any other implementation?

Reply:
I think it is affecting other modules as your non-flaky tests are failing.

Reply:
I will check this then.

Reply:
Don't we still need this in order to properly support offline bitmap assets? We should retain resource loading for binary images, and also add that support for SVG.

Reply:
I need to add support for both. But I am not getting clear idea on how I can add svg support for GlideModule along with ImageAssetFetcher.

Reply:
I tried using append function like 
      `append(ImageAssetFetcher::class.java, InputStream::class.java, RepositoryModelLoader.Factory())`. and it is working.